doc/building.md

```diff
@@ -106,8 +106,11 @@ of the `legal` directory in the generated image.
 
 ## Running `jtreg` Self-Tests
 
-The tests can be invoked with individual make targets, or collectively via the
-`test` target.
+`make test` will run all the self-tests. To invoke individual tests, use the 
+`.ok` file of each test as target, for example
+```
+bash build/make.sh -- $(pwd)/build/test/ControlTest.ok
+```
```

Comment:
Hello Ludvig, I think the idea of documenting this is a good one. I myself struggled to find a way to run individual tests until I saw your PR.

As for the proposed text, I think it will still leave open a lot of questions - like, how does one find the right `.ok` target to pass to this command. The answer to that already resides in https://github.com/openjdk/jtreg/blob/master/test/README.md#makefiles. I think as part of the current proposed change, we may have to include a pointer to that other file to provide context about the `.ok` target or maybe just migrate that README.md file contents into this building.md file.

Reply:
Thanks for the suggestion, I didn't know about that file! I've added a pointer and also borrowed the formulation from there, it sounded better IMO.

Reply:
I like your version because the technical details are more explicit.